heimdall-middleware-spec/pom.xml

```diff
@@ -60,4 +60,4 @@
 	</dependencies>
 
 	<description>This module is a specification of a Middleware structure.</description>
-</project>
\ No newline at end of file
+</project>
```

Comment:
add a blank line on the end of this - this should then remove the file from the list of changes.